osu.Server.Spectator/Database/Models/multiplayer_playlist_item.cs

```diff
@@ -19,13 +19,21 @@ public class multiplayer_playlist_item
         public long room_id { get; set; }
         public int beatmap_id { get; set; }
         public short ruleset_id { get; set; }
-        public short? playlist_order { get; set; }
+        public ushort? playlist_order { get; set; }
         public string? allowed_mods { get; set; }
         public string? required_mods { get; set; }
-        public DateTimeOffset? created_at { get; set; }
-        public DateTimeOffset? updated_at { get; set; }
+
+        /// <summary>
+        /// Read-only.
+        /// </summary>
+        public DateTimeOffset? created_at { get; set; } = DateTimeOffset.Now;
+
+        /// <summary>
+        /// Read-only.
```

Comment:
Arguably this should be updated each time we change data in a row (I don't think we're enforcing this at a mysql level as laravel handles this for us, but just outlining the expectation).

If this *has* to be readonly, probably worth adding reasoning as to why.

Reply:
It’s updated directly in the queries of `DatabaseAccess`. It’s read-only in that if you call `db.UpdatePlaylistItem(item)`, the `updated_at` value isn’t taken from the item but calculated at a query level.

Reply:
Hmm.. so it sounds like the correct behaviour then. But this xmldoc doesn't really explain that and will likely lead to confusion.

Reply:
Is it possible to make these `get` only or protect them in some other way? Or just remove the xmldoc because it's kinda implied and will be overwritten anyways.

Reply:
Will change to `Changes to this property will not be persisted to the database.`